### PR TITLE
Functions and actions as interfaces

### PR DESCRIPTION
**Pull request by HeikoTheissen**
[ODATA-1600](https://issues.oasis-open.org/browse/ODATA-1600)

### CODE REVIEW

File: vocabularies/Org.OData.Core.V1.xml [hunk 1]
```diff
@@ -558,6 +558,18 @@ Any simple identifier | Any type listed in `Validation.OpenPropertyTypeConstrain
...
 
+      <Term Name="IsInterface" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="Action Function">
+        <Annotation Term="Core.Description" String="The tagged action or function cannot be invoked, it serves only as an interface" />
+        <Annotation Term="Core.LongDescription">
+          <String>Services that want to implement the interface define an action or function with the same signature
+          or a specialized signature that replaces abstract types like `Edm.EntityType` with concrete types.
+          These function MUST be annotated with [`ImplementsInterface`](#ImplementsInterface).</String>
+        </Annotation>
+      </Term>
+      <Term Name="ImplementsInterface" Type="Edm.String" Nullable="false" AppliesTo="Action Function">
```
Comment (ralfhandl) [inline vocabularies/Org.OData.Core.V1.xml:569]: How about using a more specific type such as [QualifiedBoundOperationName](https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.Core.V1.xml#L460:~:text=%3CTypeDefinition%20Name=%22-,QualifiedBoundOperationName,-%22) or a similar new one without the "Bound" infix?

Comment (HeikoTheissen) [reply vocabularies/Org.OData.Core.V1.xml:569]: A type for exactly one usage?

File: examples/Org.OData.Core.V1.IsSignature-equivalence.md
Comment (HeikoTheissen) [inline examples/Org.OData.Core.V1.IsSignature-equivalence.md:1]: Is this a valid example for allowed differences between signature and implementation?

File: lib/transform.js [hunk 2]
```diff
@@ -72,14 +72,24 @@ fs.readdirSync(exampleFolder)
       const json = csdl.xml2json(xml, { strict: true });
       fs.writeFileSync(
         exampleFolder + example + ".json",
-        JSON.stringify(json, null, 4)
+        JSON.stringify(json, null, 4),
       );
+      if (!example.endsWith("-sample")) {
```
Comment (ralfhandl) [inline lib/transform.js:77]: Why not use a suffix such as `-vocabsample` to trigger vocab-style markdown conversion? The vocab-style markdown conversion is the new exception, so make it an exception and not the new default.

**Review by ralfhandl — CHANGES_REQUESTED** [review]

**Review by ralfhandl — APPROVED** [review]

**Review by ralfhandl — APPROVED** [review]

**Pull request closed without merging**